vdt/versionplugin/debianize/shared.py

```diff
@@ -88,9 +90,13 @@ def __init__(self, version, args, extra_args, directory):
         self.directory = directory
         self.exit_code = 0
 
+    def update_exit_code(self, code):
+        if self.exit_code == 0:
+            self.exit_code = code
+
```

Comment:
Is this because you want to catch the first exit code you encounter (below) which is other than 0?